### DISCUSSION TITLE
创建和弦的语法糖

### DISCUSSION THREAD

**Discussion by LZhaoM** [Rainbow-Dreamer/musicpy#57] [Q&A] [answered]
`a = chord('B2,F#3,A3;D4,F#3,F#4,D4,A3,D4', interval=0.125)`
`b = chord('B2[.8;.], F#3[.8;.], A3;D4[.8;.], F#3[.8;.], F#4[.8;.], D4[.8;.], A3[.8;.], D4[.8;.]')`
`>>> a.show()`
`'chord(notes=[B2, F#3, A3, D4, F#3, F#4, D4, A3, D4], interval=[1/8, 1/8, 1/8, 1/8, 1/8, 1/8, 1/8, 1/8, 1/8], start_time=0)'`
`>>> b.show()`
`'chord(notes=[B2, F#3, A3, D4, F#3, F#4, D4, A3, D4], interval=[1/8, 1/8, 0, 1/8, 1/8, 1/8, 1/8, 1/8, 1/8], start_time=0)'`
请问一下有没有更方便的办法把和弦b给创建出来，虽然有两个音符是同时弹的。
我在和弦a尝试用`;`使A3和D4同时弹出，不过行不通。
谢谢

**Comment by Rainbow-Dreamer** [comment, marked as answer]
你这样写是可以的，只是需要把参数`interval`改成`default_interval`就行了，因为interval这个参数的用法是全局设定音符间隔，所以原来设定为同时弹的音符会被这个全局的音符间隔覆盖掉，`default_interval`参数是设定默认的音符间隔，同时弹的音符就不会被影响。

**Reply by LZhaoM** [reply]
尝试了一下`default_interval`，有效果，不过我这边看说是`default_interval`是`int`类型就没办法给0.125这种`float`类型的值了。

**Comment by Rainbow-Dreamer** [comment]
> 尝试了一下`default_interval`，有效果，不过我这边看说是`default_interval`是`int`类型就没办法给0.125这种`float`类型的值了。

`default_interval`参数是不限制为`int`类型的，可以使用任意整数或者浮点数。

**Reply by LZhaoM** [reply]
谢谢，确实没被限制，应该是我这边的coc.nvim出了些问题
